src/scss/yellow/components/_header-banner.scss

```diff
@@ -0,0 +1,191 @@
+.header-banner {
+  display: block;
+  height: auto;
+  padding-bottom: $size-spacing-xxl;
+
+  .page-title-header {
+    font-weight: $font-style-weight-light;
+    line-height: $size-line-height-m;
+
+    @media screen and (min-width: $media-normal) {
+      line-height: $size-line-height-xl;
+    }
+
+    small {
+      font-size: $size-font-xs;
+      line-height: $size-font-xs;
+      margin-top: $size-spacing-xxs;
+
+      @media screen and (min-width: $media-normal) {
+        font-size: $size-font-m;
+        line-height: $size-line-height-m;
+      }
+    }
+  }
+
+  &--border {
+    border-bottom: 1px $border-style-default $color-interface-shade-dark;
+  }
+
+  &__body {
+    height: auto;
+    margin: $size-spacing-s 0 auto;
+
+
+    .container {
+      align-items: center;
+      display: flex;
+      margin: 0;
+      overflow: hidden;
+      padding: 0;
+      position: relative;
+      width: 100%;
+
+      @media screen and (min-width: $media-normal) {
+        min-height: 480px;
+        padding-left: 0;
+        padding-right: 0;
+      }
+    }
+  }
+
+  &__container-feature {
+    display: flex;
+    flex-direction: column;
+
+    &.space-box-medium {
+      margin-top: $size-spacing-xxs;
+
+      @media screen and (min-width: $media-normal) {
+        margin-top: $size-spacing-m;
+      }
+    }
+
+    @media screen and (min-width: $media-normal) {
+      flex-direction: row;
+      justify-content: space-between;
+      max-width: 600px;
+    }
+  }
+
+  &__feature {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    margin-top: $size-spacing-xxs;
+  }
+
+  &__feature-label {
+    color: $color-interface-content-medium;
+    font-style: italic;
+  }
+
+  &__badge {
+    margin-right: 10px;
+  }
+
+  &__star {
+    @media screen and (min-width: $media-normal) {
+      margin-top: 5px;
```

Comment:
Here we do not have a spacing of 5px size, should we not review this spacing to keep within the tokens?